src/helpers.js

```diff
@@ -48,3 +48,11 @@ export const convertImageUrlToCatchable = imageUrl => {
   if (isCboardProductionBlobContainer) return cboardBlobUsingCDN(imageUrl);
   return null;
 };
+
+export const resolveLabel = ({ label, labelKey }, intl) => {
```

Comment:
The name `resolveLabel` is too generic, especially for a helper util that will be imported in many files.
I suggest using something more specific, like `resolveTileLabel`.

Reply:
True!!